.gometalinter.json

```diff
@@ -0,0 +1,28 @@
+{
+    "Exclude": [
+        ".*_mock.go",
+        "geth/jail/doc.go"
+    ],
+    "Skip": ["helpers", "static", "testing", "e2e"],
+    "Vendor": true,
+    "Enable": [
```

Comment:
For all curious reviewers, `unused`, `staticcheck`, `gosimple` are part of `megacheck` so they are included.